chat-widget/src/components/livechatwidget/common/startChat.ts

```diff
@@ -228,4 +237,47 @@ const setCustomContextParams = (chatSDK: any) => {
     }
 };
 
-export { prepareStartChat, initStartChat, setPreChatAndInitiateChat };
\ No newline at end of file
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const checkIfConversationStillValid = async (chatSDK: any, props: any, requestId: any): Promise<boolean> => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let conversationDetails: any = undefined;
+
+    //For auth chat
+    if (props.getAuthToken) {
+        const authClientFunction = getAuthClientFunction(props.chatConfig);
+        if (props.getAuthToken && authClientFunction) {
```

Comment:
props.getAuthToken is validated twice, in 246 and 248

Reply:
fixed